app/models/cdr/table.rb

```diff
@@ -12,11 +12,18 @@
 #
 
 class Cdr::Table < Cdr::Base
-  extend PgCdrPartitioning
-
   self.table_name = 'sys.cdr_tables'
 
-  has_paper_trail class_name: 'AuditLogItem'
+  include PgPartitioningMixin
+
+  self.partitioned_model = Cdr::Cdr
+  self.partition_schema = 'cdr'.freeze
+  self.partition_key = :time_start
+  self.trigger_function_name = 'cdr.cdr_i_tgf'.freeze
+  self.trigger_name = 'cdr.cdr_i_tg'.freeze
```

Comment:
Let it be with default values, generate them from `self.partitioned_mode.table_name`